config/initializers/devise.rb

```diff
@@ -4,7 +4,7 @@
   # The secret key used by Devise. Devise uses this key to generate
   # random tokens. Changing this key will render invalid all existing
   # confirmation, reset password and unlock tokens in the database.
-  # config.secret_key = '90dbfa956c0deeaad5cc1683d993e0d2de68b2b81d987e10ff57556c34adf7f75b0ddd45f3d276d1c0ce7d31e4c5209756e085d01c0f4b997f3b4c1c10c49103'
+  config.secret_key = '90dbfa956c0deeaad5cc1683d993e0d2de68b2b81d987e10ff57556c34adf7f75b0ddd45f3d276d1c0ce7d31e4c5209756e085d01c0f4b997f3b4c1c10c49103'
```

Comment:
Line is too long. [152/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.